File: InMotionGIT.Common - Copy/Helpers/Directory.vb
Imports System.IO

Namespace Helpers

    Public Class Directory

        Public Shared Function GetPathRoot() As String
            Dim result As String
            result = String.Format("{0}", Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly.Location))
            Return result
        End Function

    End Class

End Namespace

File: Common/InMotionGIT.Common/Helpers/Directory.cs
using System.IO;

namespace InMotionGIT.Common.Helpers
{

    public class Directory
    {

        public static string GetPathRoot()
        {
            string result;
            result = string.Format("{0}", Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().Location));
            return result;
        }

    }

}